ISSUE_TEMPLATE.md

```diff
@@ -14,6 +14,21 @@ Large features typically go through three stages: [Alpha, Beta, and Stable](http
 Each stage requires various approvals from various teams.  Features require several releases
 to reach Stable.
 
+The following items have to be filled before the features freeze:
+
+- Primary contact (assignee);
+- SIG, responsible for the feature;
+- Proposal link - do the public design proposal in the community repo;
+- Reviewer(s) - recommend having 2+ reviewers (at least one from code-area OWNERS file) agreed to review. Reviewers from multiple companies preferred;
+- Approver (for LGTM, likely from SIG to which feature belongs);
+- Target - Alpha / beta / stable;
+- Timeline / status;
+
+Before the coding freeze:
+
+- Release note - a short sentence, describing the feature;
+- User documentation PR link (to http://github.com/kubernetes/kubernetes.github.io)
```

Comment:
I think that's unlikely to work, sadly. The timeframe is too compressed.

Another approach:

Change the current beta releases to release candidates

When we previously would have cut 1.X.0, we cut a beta release

We actively try to get people to use the beta(s)

After the inevitable couple of critical patch releases, we cut the stable release